src/adapter/src/coord/read_policy.rs

```diff
@@ -205,12 +204,12 @@ impl<T: TimelyTimestamp> ReadHolds<T> {
 impl<T: TimelyTimestamp + Lattice> ReadHolds<T> {
     pub fn least_valid_read(&self) -> Antichain<T> {
         let mut since = Antichain::from_elem(T::minimum());
-        for (_id, hold) in self.storage_holds.iter() {
-            since.join_assign(&hold.since().to_owned());
+        for hold in self.storage_holds.values() {
+            since.join_assign(hold.since());
         }
 
-        for (_id, hold) in self.compute_holds.iter() {
-            since.join_assign(&hold.frontier().to_owned());
+        for hold in self.compute_holds.values() {
+            since.join_assign(hold.since());
```

Comment:
`since()` is the frontier to which the read hold is holding back the collection's read frontier. A `frontier()` method doesn't exist on `ReadHold`! Previously the type of `hold` was `MutableAntichain`, which has a `frontier()` method but not `since()`.